consensus/src/dag/dag_store.rs

```diff
@@ -272,6 +272,10 @@ impl Dag {
     }
 
     pub fn lowest_incomplete_round(&self) -> Option<Round> {
+        if self.nodes_by_round.is_empty() {
+            return Some(self.lowest_round());
+        }
```

Comment:
If the DAG is empty you return Some(lowest_round), but if the DAG is full you return None. The question is why empty DAG is not considered full. In the sense that all rounds either have all nodes or none?

Reply:
Good point, removed the option and now I return the actual round that is incomplete. For a full dag, its highest_round+1, and an empty dag, its lowest_round.